tuxemon/core/components/player.py

```diff
@@ -698,6 +698,39 @@ def get_adjacent_tiles(self, curr_loc, game):
             adj_tiles.append((curr_loc[0]+1,curr_loc[1]))
         return adj_tiles
 
+    def add_item(self, action):

```

Comment:
Instead of passing `action` as an argument, we should just need to pass the "item to add" string. That way we can call `add_item` on a player/npc object from anywhere without having to build an action object to pass to this method.